docker/k8s/propose-requests-job.yaml

```diff
@@ -11,7 +11,8 @@ spec:
       restartPolicy: OnFailure
       containers:
       - name: request-proposer
-        image: sammyne/hyperledger-tcf
+        image: tcf-dev
```

Comment:
did you miss the TAG here?

Reply:
Yeah. I will explicitly add latest build tag

Reply:
Done